content/en/docs/reference/command-line-tools-reference/feature-gates/MonitorInformerStaleness.md

```diff
@@ -0,0 +1,14 @@
+---
+title: MonitorInformerStaleness
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta 
+    defaultValue: false
+    fromVersion: "1.35"
+---
+
+Enable metrics to watch for staleness of the KCM informer cache.
```

Comment:
Can we be more specific about to which component this gate should be applied?
Is it a `kube-controller-manager` specific gate?